tests/zipfile/test_unzip.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+import os
+
+import pytest
+
+from cookiecutter import zipfile
+
+
+def mock_download():
+    with open('tests/files/fake-repo-tmpl.zip', 'rb') as zf:
+        chunk = zf.read(1024)
+        while chunk:
+            yield chunk
+            chunk = zf.read(1024)
+
+
+def test_unzip_local_file(mocker, tmpdir):
+    """In `unzip()`, a local file reference is just unzipped where it is.
+    """
+    mock_prompt_and_delete = mocker.patch(
+        'cookiecutter.zipfile.prompt_and_delete',
+        return_value=True,
+        autospec=True
+    )
+
+    clone_to_dir = tmpdir.mkdir('clone')
+
+    output_dir = zipfile.unzip(
+        'tests/files/fake-repo-tmpl.zip',
+        is_url=False,
+        clone_to_dir=str(clone_to_dir)
+    )
+
+    assert output_dir == os.path.join(str(clone_to_dir), 'fake-repo-tmpl')
```

Comment:
Here and below, we can use the `py.path.local` api for consistency: `clone_to_dir.join('fake-repo-tmpl')`